addons/account/models/account_journal_dashboard.py

```diff
@@ -449,11 +449,11 @@ def _fill_sale_purchase_dashboard_data(self, dashboard_data):
         late_query_results = group_by_journal(self.env.cr.dictfetchall())
 
         to_check_vals = {
-            vals['journal_id'][0]: vals
-            for vals in self.env['account.move'].read_group(
+            journal: (count, amount)
+            for journal, count, amount in self.env['account.move']._read_group(
                 domain=[('journal_id', 'in', sale_purchase_journals.ids), ('to_check', '=', True)],
-                fields=['amount_total_signed'],
-                groupby='journal_id',
+                groupby=['journal_id'],
+                aggregates=['__count', 'amount_total_signed:sum'],
             )
         }
```

Comment:
```suggestion
                groupby=['journal_id'],
                aggregates=['amount_total_signed:sum', '__count'],
            )
        }
```
Small change to stay consistent with to_checks definitions of amount, count=...